modules/install-osp-deploy-dualstack.adoc

```diff
@@ -48,38 +47,95 @@ controlPlane:
 metadata:
   name: mycluster
 networking:
-  machineNetwork: <2>
+  machineNetwork: <1>
   - cidr: "192.168.25.0/24"
   - cidr: "fd2e:6f44:5dd8:c956::/64"
-  clusterNetwork: <2>
+  clusterNetwork: <1>
   - cidr: 10.128.0.0/14
     hostPrefix: 23
   - cidr: fd01::/48
     hostPrefix: 64
-  serviceNetwork: <2>
+  serviceNetwork: <1>
   - 172.30.0.0/16
   - fd02::/112
 platform:
   openstack:
-    ingressVIPs: ['192.168.25.79', 'fd2e:6f44:5dd8:c956:f816:3eff:fef1:1bad'] <3>
-    apiVIPs: ['192.168.25.199', 'fd2e:6f44:5dd8:c956:f816:3eff:fe78:cf36'] <4>
-    controlPlanePort: <5>
-      fixedIPs: <6>
-      - subnet: <7>
+    ingressVIPs: ['192.168.25.79', 'fd2e:6f44:5dd8:c956:f816:3eff:fef1:1bad'] <2>
+    apiVIPs: ['192.168.25.199', 'fd2e:6f44:5dd8:c956:f816:3eff:fe78:cf36'] <3>
+    controlPlanePort: <4>
+      fixedIPs: <5>
+      - subnet: <6>
           name: subnet-v4
           id: subnet-v4-id
-      - subnet: <7>
+      - subnet: <6>
           name: subnet-v6
           id: subnet-v6-id
       network: <7>
         name: dualstack
         id: network-id
 ----
+<1> You must specify an IP address range for both the IPv4 and IPv6 address families.
+<2> Specify the virtual IP (VIP) address endpoints for the Ingress VIP services to provide an interface to the cluster.
+<3> Specify the virtual IP (VIP) address endpoints for the API VIP services to provide an interface to the cluster.
+<4> Specify the dual-stack network details that are used by all of the nodes across the cluster.
+<5> The CIDR of any subnet specified in this field must match the CIDRs listed on `networks.machineNetwork`.
+<6> You can specify a value for either `name` or `id`, or both.
+<7> Specifying the `network` under the `ControlPlanePort` field is optional.
++
+. Alternatively if you want an IPv6 primary dual-stack cluster, edit the `install-config.yaml` file following the example below:
```

Comment:
```suggestion
. Alternatively, if you want an IPv6 primary dual-stack cluster, edit the `install-config.yaml` file following the example below:
```

Reply:
If the user performs step 4 or step 5, should they be non-numbered substeps under the same step? 

```
Perform one of the following: 
* To configure IPv4 and IPv6 address endpoints for cluster nodes....
* To configure an IPv6 primary dual-stack cluster... 

```
If they are sequential steps, maybe use _Optional:_.

Reply:
Ah - good call. They are not sequential, so I removed the numbering so that it still falls under step 4. :)